internal/service/project/resource_project.go

```diff
@@ -679,7 +679,7 @@ func SetSlowOperationThresholding(ctx context.Context, performanceAdvisorAPI adm
 func ReadIsSlowMsThresholdingEnabled(ctx context.Context, api admin.PerformanceAdvisorApi, projectID string, warnings *diag.Diagnostics) (bool, error) {
 	response, err := api.GetManagedSlowMs(ctx, projectID).Execute()
 	if err != nil {
-		if apiError, ok := admin.AsError(err); ok && *apiError.ErrorCode == "USER_UNAUTHORIZED" {
+		if apiError, ok := admin.AsError(err); ok && apiError.ErrorCode == "USER_UNAUTHORIZED" {
```

Comment:
(nit) Leo's suggestion could be used here too: `if admin.IsErrorCode(err, "USER_UNAUTHORIZED")`